todo_app/src/Data/trello_items.py

```diff
@@ -0,0 +1,92 @@
+from tokenize import String
+from typing import Iterator
+from todo_app.src.Data.item import Item
+
+import requests
+import os
+
+
+class TrelloItems:
+    def __init__(self):
+        self._boardId = os.getenv('TRELLO_BOARDID')
+        self._key = os.getenv('TRELLO_KEY')
+        self._token = os.getenv('TRELLO_TOKEN')
+        self._default_list = 'To Do'
+
+    def change_status(self, title, status):
+
+        cards = self.__get_board_cards(self._boardId)
+        cardId = None
+        for card in cards:
+            if card['name'] == title:
+                cardId = card['id']
+        if cardId == None:
+            return
+
+        lists = self.__get_board_lists(self._boardId)
+        listId = None
+        for list in lists:
+            if list['name'] == status:
+                listId = list['id']
+        if listId == None:
+            return
+        
+        self.__set_card_list(cardId, listId)
+
+
+    def get_statuses(self) -> Iterator[str]:
+        lists = self.__get_board_lists(self._boardId)
+        for list in lists:
+            yield list['name']
+
+
+    def get_items(self) -> Iterator[Item]:
+        lists = self.__get_board_lists(self._boardId)
+        for card in self.__get_board_cards(self._boardId):
+            listName = self.__get_list_name(lists, card['idList'])
+            yield Item(card['id'], listName, card['name'])
+    
+
+    def get_item(self, id) -> Item:
+        return None
+
+
+    def add_item(self, title):
+        lists = self.__get_board_lists(self._boardId)
+        listId = None
+        for list in lists:
+            if list['name'] == self.__default_list:
```

Comment:
I think this is currently broken

```suggestion
            if list['name'] == self._default_list:
```